pandas/tests/indexes/datetimes/test_datetime.py

```diff
@@ -425,3 +425,22 @@ def test_index_map(self, name):
             ((2018,), range(1, 7)), names=[name, name]
         )
         tm.assert_index_equal(index, exp_index)
+
+    def test_split_non_utc(self):
+        indices = pd.date_range("2016-01-01 00:00:00+0200", freq="S", periods=10)
+        split_indices = np.split(indices, indices_or_sections=[])
```

Comment:
Could you name these variables `result` and `expected` so the comparison below can be

`tm.assert_index_equal(result, expected)`

Reply:
are there enough functions like np.split to merit a file like tests.indexes.datetimes.test_numpy_compat?

Reply:
@mroeschke right, done.

@jbrockmendel np.split is the only function I know for which such an issue occurred. But this is my first PR for this project so maybe there are others that I haven't noticed.